relational/test_table.py

```diff
@@ -479,112 +499,119 @@ def test_project_details_permit_addenda_summary(basic_graph, d):
 def test_project_details_bedroom_info_mohcd(basic_graph, d):
     table = ProjectDetails()
 
-    entries1 = [
-        Entry('1',
-              PPTS.NAME,
-              [NameValue('residential_units_1br_net', '2', d)]),
-        Entry('2',
-              MOHCDPipeline.NAME,
-              [NameValue('num_1bd_units', '10', d)]),
-        Entry('3',
-              MOHCDInclusionary.NAME,
-              [NameValue('num_2bd_units', '5', d),
-               NameValue('num_3bd_units', '3', d)]),
-    ]
-    proj_pipeline = Project('uuid1', entries1, basic_graph)
-
-    entries2 = [
-        Entry('1',
-              PPTS.NAME,
-              [NameValue('residential_units_1br_net', '2', d)]),
-        Entry('2',
-              MOHCDInclusionary.NAME,
-              [NameValue('num_2bd_units', '5', d),
-               NameValue('num_3bd_units', '3', d)]),
+    tests = [
+        EntriesTestRow(
+            name='only pull information from pipeline even if inclusionary',
+            entries=[
+                Entry('1',
+                      PPTS.NAME,
+                      [NameValue('residential_units_1br_net', '2', d)]),
+                Entry('2',
+                      MOHCDPipeline.NAME,
+                      [NameValue('num_1bd_units', '10', d)]),
+                Entry('3',
+                      MOHCDInclusionary.NAME,
+                      [NameValue('num_2bd_units', '5', d),
+                       NameValue('num_3bd_units', '3', d)]),
+            ],
+            want={
+                'residential_units_1br': '10',
+                'residential_units_2br': '',
+                'residential_units_3br': '',
+            }),
+        EntriesTestRow(
+            name='pull info from inclusionary because no other choice',
+            entries=[
+                Entry('1',
+                      PPTS.NAME,
+                      [NameValue('residential_units_1br_net', '2', d)]),
+                Entry('2',
+                      MOHCDInclusionary.NAME,
+                      [NameValue('num_2bd_units', '5', d),
+                       NameValue('num_3bd_units', '3', d)]),
+            ],
+            want={
+                'residential_units_1br': '2',
```

Comment:
Not specifically about the test but moreso about the logic, it seems odd to me that we just add up the different data sources here. Wouldn't we want to do what we do with the MOHCD data sources and be internally consistent (so use only one data source)?

Reply:
you're right. i added the "only one mohcd" logic later on and never updated this part.

Reply:
that would be a cleaner way to handle the uniqueness aspect

Reply:
that would be a cleaner way to handle the uniqueness aspect